bluechi.spec.in

```diff
@@ -203,7 +203,7 @@ API description and manually written code to simplify recurring tasks.
 %autosetup
 
 %build
-%meson -Dapi_bus=system
+%meson -Dapi_bus=system -Dset_coverage=true
```

Comment:
this should be under a conditional. You can see how to do that here: https://github.com/containers/bluechi/pull/443/files

Reply:
Please move all spec changes into the following commit, in this commit we only need a way to enable/disable building with coverage support for meson